src/components/TypeBase/TypeBase.js

```diff
@@ -4,26 +4,26 @@ import PropTypes from 'prop-types'
 import useIncludes from '../../hooks/useIncludes.js'
 import useInvalid from '../../hooks/useInvalid.js'
 import { COLORS } from '../Colors'
-import styles from './Type.module.scss'
+import styles from './TypeBase.module.scss'
 
-/* @getethos/design-system/Type.js
+/* @getethos/design-system/TypeBase.js
 
    Legend:
 
-   - `Type` is a private component that returns an element with CSS classes.
-   - `Type.module.scss` styles the element via the classes.
+   - `TypeBase` is a private component that returns an element with CSS classes.
+   - `TypeBase.module.scss` styles the element via the classes.
    - `TypeFoundry` is a HOC that creates public components with correct props.
    - `<Caption.Medium500>`, etc. are the Design-approved public components.
    ========================================================================== */
 
 /**
- * `Type` is a private component that returns an element with CSS classes.
+ * `TypeBase` is a private component that returns an element with CSS classes.
```

Comment:
I went with `TypeBase` instead of `BaseType`, so it sits next to `Type` and `TypeTwo`.

Reply:
I dig it